packages/core/src/html/NodeProcessor.js

```diff
@@ -597,141 +140,85 @@ class NodeProcessor {
-        this.docId += 1; // used in markdown-it-footnotes
+        this.markdownProcessor.docId += 1; // used in markdown-it-footnotes
         return processInclude(node, context, this.pageSources, this.variableProcessor,
-                              text => this._renderMd(text), text => this._renderMdInline(text),
+                              text => this.markdownProcessor.renderMd(text),
+                              text => this.markdownProcessor.renderMdInline(text),
                               this.config);
       case 'panel':
-        this._processPanelAttributes(node);
+        this.componentProcessor.processPanelAttributes(node);
         return processPanelSrc(node, context, this.pageSources, this.config);
       case 'question':
-        this._processQuestion(node);
+        this.componentProcessor.processQuestion(node);
         break;
       case 'q-option':
-        this._processQOption(node);
+        this.componentProcessor.processQOption(node);
         break;
       case 'quiz':
-        this._processQuiz(node);
+        this.componentProcessor.processQuiz(node);
         break;
       case 'popover':
-        this._processPopover(node);
+        this.componentProcessor.processPopover(node);
+        transformBootstrapVuePopover(node);
         break;
       case 'tooltip':
-        this._processTooltip(node);
+        this.componentProcessor.processTooltip(node);
+        transformBootstrapVueTooltip(node);
         break;
       case 'modal':
-        this._processModalAttributes(node);
+        this.componentProcessor.processModalAttributes(node);
+        transformBootstrapVueModalAttributes(node,
+                                             (originalName, newName) =>
+                                               NodeProcessor._renameSlot(node, originalName, newName),
+                                             (originalAttribute, newAttribute) =>
+                                             // eslint-disable-next-line max-len
+                                               NodeProcessor._renameAttribute(node, originalAttribute, newAttribute));
         break;
       case 'tab':
       case 'tab-group':
-        this._processTabAttributes(node);
+        this.componentProcessor.processTabAttributes(node);
         break;
       case 'box':
-        this._processBoxAttributes(node);
+        this.componentProcessor.processBoxAttributes(node);
         break;
       case 'dropdown':
-        this._processDropdownAttributes(node);
+        this.componentProcessor.processDropdownAttributes(node);
         break;
       case 'thumbnail':
-        this._processThumbnailAttributes(node);
+        this.componentProcessor.processThumbnailAttributes(node);
         break;
       case 'annotation':
-        NodeProcessor._processAnnotationAttributes(node);
+        // Annotations are added automatically by KaTeX when rendering math formulae.
+        if (!_.has(node.attribs, 'v-pre')) { node.attribs['v-pre'] = true; }
```

Comment:
let's fallthrough with the one for `<code>`

Reply:
Okay I have made the modifications for this